modules/ROOT/partials/configuration/tinycomments_resolve.adoc

```diff
@@ -7,7 +7,7 @@ The `+tinycomments_resolve+` function should asynchronously return a flag indica
 
 The `+tinycomments_resolve+` function is passed a (`+req+`) object as the first parameter, which contains the following key-value pair:
 
-`+conversationUid+`:: The uid of the conversation the reply is targeting.
+* `+conversationUid+`: The uid of the conversation the reply is targeting.
```

Comment:
```suggestion
* `+conversationUid+`: The uid of the targeting conversation
```

???

Reply:
I feel like `The uid of the conversation the reply is targeting.` is more specific here.

Reply:
After investigation and reviewing with @tiny-ben-tran  this statement is correct: `The uid of the targeting conversation`.